.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,48 @@
+# Work Item Issue Template
+description:
+title: "[Work_Item ]"
+labels: ["work item"]
+assignees: [shawnalpay, jpradocueva]
+## 1. **Problem Statement** *
+Describe the problem, issue, use case, or opportunity that this work item addresses.
+- **What is the problem?**: Explain the context and why it needs resolution.
+- **Impact**: Describe how the problem affects users, systems, or the project.
+
+Your input ...
+
+## 2. **Objective** *
+State the objective of this work item. What outcome is expected?
+- **Success Criteria**: Define how success will be measured (e.g. columns, KPIs, metrics, content).
+
+Your input ...
+
+## 3. **Supporting Documentation** *
+Include links to supporting documents such as:
+- Data Examples: [Link to data or relevant files]
+- Related Use Cases or Discussion Documents: [Link to discussion]
+- PRs or Other References: [Link to relevant references]
+
+Your input ...
+
+## 4. **Proposed Solution / Approach**
+Outline any proposed solutions, approaches, or potential paths forward.
+- **Initial Ideas**: Describe potential solution paths, tools, or technologies.
+- **Considerations**: Include any constraints, dependencies, or risks.
```

Comment:
include preliminary feasibility assessment:
What will it take to implement?
What is the size of lift to align on spec?
Can this be done by mapping data from existing CSP exports?

Reply:
Although we will leave formal design and development to the spec development portion of the release cycle -- I agree it would of course be good to capture as much info as possible at this stage.  How about:
```suggestion
- **Considerations**: Include any constraints, dependencies, or risks.
- **Feasibility**: Include any information that helps quantify feasibility, such as perceived level of effort or existing fields in current data generator exports.
```

Reply:
Is this the right phase to have a feasibility or effort sizing. This is both a concern on friction and also the person submitting may not know the feasibility or size.